### PR TITLE
Adapt options to new Request model & some cleanup

### PR DESCRIPTION
**Pull request by dbengsch**

### CODE REVIEW

File: source/includes/_kuzzleDataCollection.md [hunk 1]
```diff
@@ -1067,7 +1067,7 @@ catch (ErrorException $e) {
 
 ```json
 { "total": 3,
-  "documents": [{object}, {object}, {object}]
+  "documents": [{"some": "document"}, {"some": "document"}, {"some": "document"}]
```
Comment (scottinet) [inline source/includes/_kuzzleDataCollection.md]: Shouldn't this be: `"documents": [<KuzzleDocument>, <KuzzleDocument>, <KuzzleDocument>]`?

File: source/includes/_kuzzleDataCollection.md [hunk 2]
```diff
@@ -1802,9 +1802,9 @@ catch (ErrorException $e) {
 
 ```json
 { "total": 3,
-  "documents": [{object}, {object}, {object}],
+  "documents": [{"some": "document"}, {"some": "document"},{"some": "document"}],
```
Comment (scottinet) [inline source/includes/_kuzzleDataCollection.md]: Shouldn't this be: `"documents": [<KuzzleDocument>, <KuzzleDocument>, <KuzzleDocument>]`?

File: source/includes/security/_kuzzleSecurity.md [hunk 3]
```diff
@@ -2143,12 +2146,12 @@ catch (ErrorException $e) {
 
 > Callback response:
 
-```js
+```json
 {
-  total: 124,
-  documents: [
+  "total": 124,
+  "documents": {
```
Comment (scottinet) [inline source/includes/security/_kuzzleSecurity.md]: `documents` is an array

File: source/includes/security/_kuzzleSecurity.md [hunk 4]
```diff
@@ -2309,12 +2312,12 @@ catch (ErrorException $e) {
...
```
Comment (scottinet) [inline source/includes/security/_kuzzleSecurity.md]: `documents` is an array

**Comment by dbengsch** [comment]
@scottinet Done

**Pull request merged**